docs/sphinx/index.rst

```diff
@@ -73,6 +73,16 @@ OME-XML
     ome-tiff/tools
     ome-xml/java-library
 
+**************
+OMERO Pyramids
```

Comment:
Maybe make singular "Pyramid" for consistency with the other things in the index.